docs/pages/machine-id/troubleshooting.mdx

```diff
@@ -292,3 +292,31 @@ flag:
 $ tctl bots rm example
 $ tctl bots add example --roles=foo,bar,machine-id-db
 ```
+
+## Destination kubernetes_secret: `identity-output` must be a directory in exec plugin mode
+
+By default, when outputting a Kubernetes identity, `tbot` outputs make use of a Kubernetes exec
+plugin to always provide the "latest" version of the credentials.
```

Comment:
What do the scare quotes mean here? Would the sentence mean the same thing without them?

Reply:
Yes, good point.